contributor-guide/modules/ROOT/pages/getting-involved.adoc

```diff
@@ -1,4 +1,60 @@
+
+== Report Issues and Suggest Improvements
+
+To formally report an issue on an individual library, locate the repo for that library. Many of these repos exist in the https://github.com/cppalliance/boost/tree/master/libs[Boost Super-project] on GitHub. Search under the *libs* folder. For example, to file an issue on the boost:asio[] library, click *New Issue* at https://github.com/boostorg/asio/issues[]. If you have difficulty locating the correct repo, then ask the question of the https://lists.boost.org/mailman/listinfo.cgi/boost[Boost developers' mailing list].
+
+
+To report an issue with the web site, including this documentation, click the *New Issue* button on this GitHub page: https://github.com/cppalliance/site-docs/issues[cppalliance/site-docs]. 
+
+After clicking *New Issue*, you will see a list of issue templates, such as *Submit a Correction or Bug report*, or *Suggest a Topic, Enhancement, or Information*. Select the most appropriate template (which will give you some advice on how to fill out the form), or select *Open a blank issue* if none of the templates applies.
+
+Whichever list you add your issue to, the issue will be addressed in due course. Providing full details, and examples and screenshots where appropriate, should give you the quicker response.
+
+== Contribute to an Existing Library
+
+Rather than report an issue, or suggested improvement, and wait for another developer to address it, you can make the code changes yourself. To be successful you will need to be familiar with GitHub, and obviously a competent pass:[C++] developer. After finding the issue, or coming up with the improvement, you will need to go to the GitHub repository for that library, and create a fork and clone the library.
+
+With your fork of the library in your own GitHub account, you can make any changes and updates you wish. It is certainly recommended to thoroughly test the changes, and be aware of the test matrix for the library. You may have to test your changes on several operating systems, and with more than one compiler. It is also recommended you thoroughly comment your code changes, potentially every line of code you add should be commented as to its purpose.
+
+When you are satisfied you have met the bar for testing, you can commit the changes, push them to GitHub, and create a Pull Request. 
+
+After this, you can take a step back, and the owners or maintainers of the library will review your code. They might ask for some modifications or improvements. Incorporate the feedback provided by the reviewers and push your modifications.
+
+If all goes well, your changes will be merged by the library admins, and you will have made a welcome contribution.
+
+== Engage in the Review Process
+
+There are several roles in the review process, from managing the steps of the review process, to testing and providing feedback. Read the xref:formal-reviews:ROOT:index.adoc[] for details.
```

Comment:
If I were a user reading this, I'd be missing some context - what review process? It may be worth adding a generic sentence about how Boost libraries get reviewed by the community before becoming part of Boost.

Reply:
OK, added an intro sentence.